invenio_drafts_resources/services/records/service.py

```diff
@@ -293,14 +301,7 @@ def update_draft(
         # Commit and index
         uow.register(RecordCommitOp(draft, indexer=self.indexer))
 
-        current_audit_logs_service.create(
-            data=dict(
-                action="draft.edit",
-                resource=dict(type="record", id=str(id_)),
-            ),
-            identity=identity,
-            uow=uow,
-        )
+        uow.register(AuditLogOp(DraftEditAuditLog.build(id_, identity), identity))
```

Comment:
```suggestion
        uow.register(AuditLogOp(DraftEditAuditLog.build(identity, id_)))
```

We should only keep the `identity` of the "actor" of the audit log event. The identity that creates the audit log is anyways always the system identity.